lib/boilerplate.js

```diff
@@ -49,6 +49,11 @@ const DEFAULT_OPTS = {
     owner: 'appium',
     repo: 'appium-build-store',
   },
+  yamllint: true,
+  yaml: {
+    files: ['./.travis.yml'],
```

Comment:
shall we include all files with yml extensions (except of these in test folders) ?

Reply:
Sure.